client/src/components/History/Content/ContentItem.vue

```diff
@@ -0,0 +1,202 @@
+                    <b-button
+                        v-if="writeable && state != 'discarded'"
+                        :disabled="isUnavailable"
+                        class="px-1"
+                        title="Edit attributes"
+                        size="sm"
+                        variant="link"
+                        @click.stop="onEdit">
+                        <span class="fa fa-pencil" />
+                    </b-button>
+                    <b-button
+                        v-if="writeable && item.deleted"
+                        class="px-1"
+                        title="Undelete"
+                        size="sm"
+                        variant="link"
+                        :disabled="isUnavailable"
+                        @click.stop="$emit('undelete', item)">
+                        <span class="fa fa-trash-restore" />
+                    </b-button>
+                    <b-button
+                        v-else-if="writeable"
+                        class="px-1"
+                        title="Delete"
+                        size="sm"
+                        variant="link"
+                        :disabled="isUnavailable"
+                        @click.stop="$emit('delete', item)">
+                        <span class="fa fa-trash" />
+                    </b-button>
+                    <b-button
+                        v-if="writeable && !item.visible"
+                        class="px-1"
+                        title="Unhide"
+                        size="sm"
+                        variant="link"
+                        @click.stop="$emit('unhide', item)">
+                        <span class="fa fa-unlock" />
+                    </b-button>
+                </div>
+                <h5 class="float-left p-1 w-75 font-weight-bold">
+                    <div v-if="selectable" class="selector float-left mr-2">
+                        <span
+                            v-if="selected"
+                            class="fa fa-lg fa-check-square-o"
+                            @click.stop="$emit('update:selected', false)" />
+                        <span v-else class="fa fa-lg fa-square-o" @click.stop="$emit('update:selected', true)" />
+                    </div>
+                    <span :class="icon" />
+                    <span class="id hid">{{ id }}</span>
+                    <span>:</span>
+                    <span class="content-title name">{{ name }}</span>
+                    <h6 v-if="item.collection_type" class="description py-1">
+                        a {{ item.collection_type }} with {{ item.element_count }} items
+                    </h6>
+                    <div v-if="!expanded && item.tags && item.tags.length > 0" class="nametags">
+                        <Nametag v-for="tag in item.tags" :key="tag" :tag="tag" />
+                    </div>
+                </h5>
+            </div>
+        </div>
+        <DatasetDetails v-if="expanded" @edit="onEdit" :item="item" />
```

Comment:
```suggestion
        <-- collections are not expandable, so we only need the DatasetDetails component here -->
        <DatasetDetails v-if="expanded" @edit="onEdit" :item="item" />
```
since I was wondering about that at first 😆